core/kover/learning/set_covering_machine/scm.py

```diff
@@ -81,7 +81,7 @@ def fit(self, rules, rule_classifications, positive_example_idx, negative_exampl
             rule_blacklist = np.unique(rule_blacklist)
             if len(rule_blacklist) == rule_classifications.shape[1]:
                 raise ValueError("The blacklist cannot include all the rules.")
-            logging.debug("The following rules are blacklisted and will not be considered:" + str(rule_blacklist))
+            logging.debug("Blacklisting : " + str(len(rule_blacklist)/2) + " kmers (" + str(len(rule_blacklist)) + " rules)")
```

Comment:
A cleaner way to write this would be:
`logging.debug("Blacklisting: {0:d} kmers ({1:d} rules)".format(len(rule_blacklist) / 2, len(rule_blacklist)))`